src/main/java/Array12.java

```diff
@@ -0,0 +1,25 @@
+public class Array12 {
+
+    public int[] insertInArray(int[] arr, int insertTo, int value) {
+        int length = arr.length;
+        int[] arrTemp = new int[length + 1];
+
+        System.arraycopy(arr, 0, arrTemp, 0, length);
+        for (int i = length; i > insertTo; --i) {
```

Comment:
```java
System.arraycopy(arr, insertTo, arrTemp, insertTo + 1, length - insertTo)
```

Reply:
ý đồ của em là muốn tạo một cái mảng in hệt với arr để thao tác dữ liệu trên nó, nếu dùng câu của anh thì không copy qua hết được. Em không hiểu ý của anh

Reply:
Nhưng mà câu lệnh đó ý nghĩa giống y chang câu em đã implement mà :|

Reply:
ok rồi anh